graph-tutorial/src/webparts/graphTutorial/GraphTutorialWebPart.ts

```diff
@@ -60,53 +61,38 @@ export default class GraphTutorialWebPart extends BaseClientSideWebPart<IGraphTu
                 </div>
               </div>
             </div>`;
-
             if (error) {
               this.renderGraphError(error);
             } else {
               this.renderCalendarView(events.value);
             }
-          });
-      });
+          })
+          .catch((reason: Error) => {
+            this.renderGraphError(reason);
+          });  
+      })
+      .catch((reason: Error) => {
+        this.renderGraphError(reason);
+      });  
   }
-  // </renderSnippet>
-  */
 
-  // <alternateRenderSnippet>
-  public render(): void {
```

Comment:
This needs to also be in the code as an alternative. This is a bit of an odd scenario, but the idea here is to show how to both render the results yourself, or use the Microsoft Graph Toolkit components to render for you.